tests/conftest.py

```diff
@@ -455,6 +457,31 @@ def run_before_and_after_tests_2(request, mapdl):
     assert prev == mapdl.is_local
 
 
+@pytest.fixture(autouse=True, scope="function")
+def run_before_and_after_tests_2(request, mapdl):
+    """Make sure we leave no MAPDL running behind"""
+    from ansys.mapdl.core.cli.stop import is_ansys_process
+
+    yield
+
+    for proc in psutil.process_iter():
+        if (
+            psutil.pid_exists(proc.pid)
+            and proc.status() in PROCESS_OK_STATUS
```

Comment:
``PROCESS_OK_STATUS`` doesn't seem to be defined according to [CICD](https://github.com/ansys/pymapdl/actions/runs/9857260057/job/27216120790)

Reply:
fixed.